packages/config/src/options.ts

```diff
@@ -181,260 +178,245 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
       specPattern: defaultSpecPattern.e2e,
     },
     validation: isValidConfig,
-    canUpdateDuringTestTime: false,
   }, {
     name: 'env',
     defaultValue: {},
     validation: validate.isPlainObject,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'execTimeout',
     defaultValue: 60000,
     validation: validate.isNumber,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'experimentalFetchPolyfill',
     defaultValue: false,
     validation: validate.isBoolean,
     isExperimental: true,
-    canUpdateDuringTestTime: false,
   }, {
     name: 'experimentalInteractiveRunEvents',
     defaultValue: false,
     validation: validate.isBoolean,
     isExperimental: true,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'experimentalSessionAndOrigin',
     defaultValue: false,
     validation: validate.isBoolean,
     isExperimental: true,
-    canUpdateDuringTestTime: false,
   }, {
     name: 'experimentalModifyObstructiveThirdPartyCode',
     defaultValue: false,
     validation: validate.isBoolean,
     isExperimental: true,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'experimentalSourceRewriting',
     defaultValue: false,
     validation: validate.isBoolean,
     isExperimental: true,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'fileServerFolder',
     defaultValue: '',
     validation: validate.isString,
     isFolder: true,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'fixturesFolder',
     defaultValue: 'cypress/fixtures',
     validation: validate.isStringOrFalse,
     isFolder: true,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'excludeSpecPattern',
     defaultValue: (options: Record<string, any> = {}) => options.testingType === 'component' ? ['**/__snapshots__/*', '**/__image_snapshots__/*'] : '*.hot-update.js',
     validation: validate.isStringOrArrayOfStrings,
-    canUpdateDuringTestTime: true,
```

Comment:
looking at this....why can this update at test time?

Reply:
I don't think it should be able to - probably incorrect.

Reply:
thats funny, what would overriding this even do?